__tests__/regression/fix-2414.test.js

```
@@ -0,0 +1,62 @@
+// Test fix of #2414: Extra clones in infinite mode when there is only one slide or unslick is true
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
```

ESLint

[error] 'fireEvent' is defined but never used.

(no-unused-vars)